CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ if(NOT multiConfig AND NOT DEFINED CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build.")
 endif()
 
-project("OpenMS_host" LANGUAGES CXX)
+project("OpenMS_host" LANGUAGES CXX C)
```

Comment:
currently needed because of the hdf5 C header.

Reply:
Yes not so bad. I removed it some time ago because it was not needed for our simple tests.